app/javascript/mastodon/features/onboarding/follows.jsx

```diff
@@ -1,79 +1,59 @@
-import PropTypes from 'prop-types';
-import { PureComponent } from 'react';
+import { useEffect } from 'react';
 
 import { FormattedMessage } from 'react-intl';
 
-import ImmutablePropTypes from 'react-immutable-proptypes';
-import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { useDispatch, useSelector } from 'react-redux';
 
 import { fetchSuggestions } from 'mastodon/actions/suggestions';
 import { markAsPartial } from 'mastodon/actions/timelines';
-import Column from 'mastodon/components/column';
 import { ColumnBackButton } from 'mastodon/components/column_back_button';
 import { EmptyAccount } from 'mastodon/components/empty_account';
 import Account from 'mastodon/containers/account_container';
 
-const mapStateToProps = state => ({
-  suggestions: state.getIn(['suggestions', 'items']),
-  isLoading: state.getIn(['suggestions', 'isLoading']),
-});
-
-class Follows extends PureComponent {
-
-  static propTypes = {
-    onBack: PropTypes.func,
-    dispatch: PropTypes.func.isRequired,
-    suggestions: ImmutablePropTypes.list,
-    isLoading: PropTypes.bool,
-  };
+export const Follows = () => {
+  const dispatch = useDispatch();
+  const [isLoading, suggestions] = useSelector(state => [state.getIn(['suggestions', 'isLoading']), state.getIn(['suggestions', 'items'])]);
```

Comment:
Everywhere else in the code (even when not using TypeScript) we use `useAppSelector` from `mastodon/store` as this is typed for our specific store. We should probably do the same here.

Also, I'm not sure what's the most efficient or idiomatic between this pattern, and doing two `useSelector` calls.

Reply:
Redux authors recommend doing two `useSelector` calls, rather than returning a complex state